src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -735,6 +735,55 @@ void CheckReLU6Correct(const arma::colvec input,
   }
 }
 
+/**
+ * Implementation of the ISRU activation function test. The function is
+ * implemented as ISRU layer in the file isru.hpp.
+ *
+ * @param input Input data used for evaluating the ISRU activation function.
+ * @param target Target data used to evaluate the ISRU activation.
+ */
+
+void CheckISRUActivationCorrect(const arma::colvec input,
+                                 const arma::colvec target)
```

Comment:
```suggestion
void CheckISRUActivationCorrect(const arma::colvec input,
                                const arma::colvec target)
```

Should align with the first parameter.

Reply:
Alright, I'll make the above mentioned changes. Thank you!